gnomad_constraint/constraint/constraint_pipeline.py

```diff
@@ -70,8 +129,34 @@ def main(args):
     parser.add_argument(
         "--overwrite", help="Whether to overwrite output files", action="store_true"
     )
+    parser.add_argument(
+        "--use-pop",
```

Comment:
I added an option for whether to analyze population because: 1) I feel that population analysis is optional when we generate constraint table. 2) The downsampling in population has the potential risk to break the pipeline. instead of having an option for downsampling, I would make population analysis optional.

Reply:
These variables will be used across multiple processes. Isn't it more clear to keep all the variable at the top and the process argument at the bottom?